packages/react-components/src/components/kpi/tests/kpi.spec.tsx

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { KPI } from '../kpi';
+import {
+  DATA_STREAM,
+  ERROR_DATA_STREAM,
+  ERROR_QUERY,
+  FILLED_THRESHOLD,
+  LOADING_QUERY,
+  SUCCESS_QUERY,
+  THRESHOLD,
+  VIEWPORT,
+} from './mockData';
+
+describe('default kpi widget', () => {
```

Comment:
I would add a few tests where the `useViewport` hook return value is changed to show how https://github.com/awslabs/iot-app-kit/blob/main/packages/react-components/src/components/kpi/kpi.tsx#L41 works.